src/elements/content-sidebar/activity-feed/comment/RepliesToggle.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import PlainButton from '../../../../components/plain-button';
+
+import './RepliesToggle.scss';
+
+type Props = {
+    onHideReplies: (index: number) => void,
+    onShowReplies: () => void,
+    repliesShownCount: number,
+    repliesTotalCount: number,
+};
+
+const RepliesToggle = ({ onShowReplies, onHideReplies, repliesShownCount, repliesTotalCount }: Props) => {
+    if (repliesTotalCount <= 1) {
```

Comment:
shouldn't the parent control whether or not this component is visible or not?

Reply:
In this case, I think checking if the count passes edgecase criteria makes sense. I guess the parent can check if repliesTotalCount is greater than 0, but shouldn't that be a given for rendering the show/reply toggle? This is checking for edgecases, which I don't think the parent should control.

Reply:
If the logic for hide/show was in the parent then it would make sense to move the conditional. But since this component is not a dumb component, we should have some guards for the logic